lib/matplotlib/rcsetup.py

```diff
@@ -951,484 +949,467 @@ def _validate_linestyle(ls):
                          "sequence.".format(ls))
 
 
-validate_axes_titlelocation = ValidateInStrings('axes.titlelocation', ['left', 'center', 'right'])
+validate_axes_titlelocation = ValidateInStrings(
+    'axes.titlelocation', ['left', 'center', 'right'])
 
-# a map from key -> value, converter
-defaultParams = {
-    'backend':           [_auto_backend_sentinel, validate_backend],
-    'backend_fallback':  [True, validate_bool],
-    'webagg.port':       [8988, validate_int],
-    'webagg.address':    ['127.0.0.1', validate_webagg_address],
-    'webagg.open_in_browser': [True, validate_bool],
-    'webagg.port_retries': [50, validate_int],
-    'toolbar':           ['toolbar2', validate_toolbar],
-    'datapath':          [None, validate_any],  # see _get_data_path_cached
-    'interactive':       [False, validate_bool],
-    'timezone':          ['UTC', validate_string],
+
+# Mapping of rcParams to validators.  The rcParams defaults are defined in
+# matplotlibrc.template, which gets copied to matplotlib/mpl-data/matplotlibrc
+# by the setup script.
+_validators = {
+    'backend':          validate_backend,
```

Comment:
I'm a bit confused over whether you want to align or not the values in this dictionary. Sometimes they are (in a block), but sometimes not.

Reply:
This was inherited from the earlier inconsistent indenting, but sure, I indented everything blockwise.